### PR TITLE
Add kube-monitoring-compute-pluginpreset

### PR DESCRIPTION
**Pull request by max-len**

### CODE REVIEW

**Review by richardtief — CHANGES_REQUESTED** [review]
outdated comment.

File: system/greenhouse-ccloud/templates/kube-monitoring-compute-pluginpreset.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  namespace: {{ .Release.Namespace }}
+data:
+  tls.crt: {{ required ".Values.kubeMonitoring.ssoCert missing" .Values.kubeMonitoring.ssoCert | b64enc }}
+  tls.key: {{ required ".Values.kubeMonitoring.ssoKey missing" .Values.kubeMonitoring.ssoKey | b64enc }}
+
+---
+apiVersion: greenhouse.sap/v1alpha1
+kind: PluginPreset
+metadata:
+  name: kube-monitoring
```
Comment (richardtief) [inline system/greenhouse-ccloud/templates/kube-monitoring-compute-pluginpreset.yaml:16]:
```suggestion
  name: kube-monitoring-compute
```

**Review by richardtief — CHANGES_REQUESTED** [review]
Solved.

**Comment by auhlig** [comment]
For now, I solved it with the existing `kube-monitoring` pluginpreset. An additional `kube-monitoring-compute` would only be necessary if its configuration significantly deviates from the `kube-monitoring` one

**Review by richardtief — APPROVED** [review]
Updated the PluginPreset accordingly.

**Pull request merged**